### ISSUE TITLE
PanelFloppy.cpp  ....GUI and helptext formatting    [spawn of commit a27b3b3]

### ISSUE THREAD

**Issue by giantclambake** [BlitterStudio/amiberry#1094]
Firstly a couple of things I noted about the floppy panel GUI ....

The Info (?) and Eject buttons should rightfully remain ghosted/inactive until a floppy image is loaded. With empty floppy slots these buttons are currently active no-ops. In a perfect world, these no-ops should pop a warning tantamount to 'no drive image inserted' if they are clicked when that is the case....this forms an 'operational hint' as such, and justifies their active state. As that's a bit of work, it's easier to make the buttons ghosted/inactive when they can serve no possible function (no image loaded).

The placement of the 3 buttons 'Save config for disk'  && 'Create 3.5" DD disk'  &&  'Create 3.5" HD disk' could give rise to the inference that these buttons pertain to every floppy device depicted on this panel (including drawbridge driver), and likewise the floppy drive emulation speed might also be misconstrued to apply to the drawbridge device itself (as it's emulating a floppy drive device =)

The reality of course is much different ~ the first button only pertains to DF0: and the other 2 buttons pertain to creating disk image files...ie; you can't do anything with drawbridge using these buttons.

IMHO the 3 buttons should be moved up so as to appear just under the floppy drive emulation speed widget, and all the drawbridge configurata should be boxed at the bottom of the panel, to differentiate those hardware device settings, from the disk image operations above it.

Somehow, aesthetically speaking, the 'Save config for disk' button, should appear closer to/adjacent to the DF0: drive settings. That would allow a wording change ...ie; "Save this disk conf"  or similar...

An error was introduced @ commit a27b3b3 line:813 but as I mentioned previously, most of the helptext areas need some TLC, and I'm happy to do that English text housekeeping.

By rights, helptext here should touch upon drawbridge options as presented by the GUI, however I didn't include that text here because of the above concerns about GUI buttons/item placement,  and the helptext window itself likely needs a scrollbar to include all required text (...or not, I should look again..it's close)...

Here's my suggested helptext corrections for PanelFloppy.cpp

```

<------>helptext.clear();
<------>helptext.emplace_back("You can enable/disable each drive by clicking the checkbox next to DFx or by selecting");
<------>helptext.emplace_back("the \"Disabled\" drive type in the dropdown menu control. The selection of the  3.5\" DD ");
<------>helptext.emplace_back("drive type is the right choice for nearly all ADF and ADZ floppy image files.");
<------>helptext.emplace_back(" ");
<------>helptext.emplace_back("The option \"Write-protected\" indicates if the emulator can write to the ADF or not.");
<------>helptext.emplace_back("Changing the write protection of the disk file may fail because of missing rights");
<------>helptext.emplace_back("on the host filesystem.");
<------>helptext.emplace_back(" ");
<------>helptext.emplace_back("The \"...\" button opens a dialog to select the desired floppy disk file to be loaded.");
<------>helptext.emplace_back("With the dropdown menu control, you can select one of the disk files most recently used.");
<------>helptext.emplace_back("Details of the currently loaded floppy file can be displayed with the \"?\" button.");
<------>helptext.emplace_back(" ");
<------>helptext.emplace_back("You can reduce the loading time for lot of games by increasing the floppy drive emulation");
<------>helptext.emplace_back("speed. A few games will not load with higher drive speed and you have to select 100%.");
<------>helptext.emplace_back(" ");
<------>helptext.emplace_back("The \"Save config for disk\" button will create a new configuration file with the name of");
<------>helptext.emplace_back("the disk image currently in DF0. If you enable the \"Load config with same name as disk\" ");
<------>helptext.emplace_back("option, this configuration file will be used each time this floppy disk image is selected.");
<------>helptext.emplace_back(" ");
<------>helptext.emplace_back("You can also use the 'Create 3.5\" DD disk' and 'Create 3.5\" HD disk' buttons, to make");
<------>helptext.emplace_back("new and empty disk images for use with the emulator.");
<------>return true;

```

HTH

**Comment by midwan** [comment]
I've updated a few things in the Floppy and Quickstart panel:
- The Eject and Info buttons are now only enabled if a disk is inserted in that drive
- Removed the default `df0.adf`, `df1.adf`, `df2.adf` and `df3.adf` images that were set as "inserted" by default. Now there is no default image in any drive at all. This was coming from WinUAE and it was meant to help auto-load a disk image if found.
- Removed the Save config and Load config from disk. This option is kind of obsolete, as you can normally do the same from the relevant Config panel, instead. Removing them reduces the clutter and frees up space to better organize the rest of the elements
- Grouped the Drawbridge options together in a visible widget
- Updated the Help text accordingly, to also include the Drawbridge options

**Comment by giantclambake** [comment]
Confirming this fix for #1094, TY ~ we have of course 'nixed 2 options here, so the corresponding wiki page needs updating...

https://github.com/BlitterStudio/amiberry/wiki/GUI:-Floppy
